src/orchard/java/legacy_parser.clj

```diff
@@ -69,7 +68,6 @@
   (when-let [res (io/resource path)]
     (let [access   (ModifierFilter. ModifierFilter/ALL_ACCESS)
           context  (doto (Context.) (Messager/preRegister "orchard-javadoc"))
-          options  (doto (Options/instance context) (.put "ignore.symbol.file" "y"))
```

Comment:
It seems to me this might have been invoked for its side effects, not for the result. Notice the `.put`.

Reply:
On the other hand if those options are not used anywhere - perhaps they are not really needed. @jeffvalk is the expert on this part of the code.